addons/sale_crm/wizard/crm_make_sale.py

```diff
@@ -113,6 +113,7 @@ def makeOrder(self, cr, uid, ids, context=None):
                 sale_order = sale_obj.browse(cr, uid, new_id, context=context)
                 case_obj.write(cr, uid, [case.id], {'ref': 'sale.order,%s' % new_id})
                 new_ids.append(new_id)
+                cr.execute('insert into quotation_sale_order_rel (lead_id,order_id) values (%s,%s)', (case.id, new_id))
```

Comment:
You're probably right. I followed the way the "View Invoices" button is implemented on the Sales Orders. 
https://github.com/odoo/odoo/blob/8.0/addons/sale/sale.py#L549
I will check it asap.